crux-test/test/crux/tx_test.clj

```diff
@@ -638,14 +638,14 @@
                (api/entity (api/db *api*) :foo))))))
 
 (t/deftest map-ordering-362
-  (t/testing "cas is independent of map ordering"
+  #_(t/testing "cas is independent of map ordering"
```

Comment:
Why commented?

Reply:
ah, because it was only the bottom part of the test that was breaking, so I commented the top two out to focus on the breaking part (so that I didn't get too many logs printed)

will uncomment :)